nf_core/pipeline-template/.gitpod.yml

```diff
@@ -0,0 +1 @@
+image: nf-core/gitpod
```

Comment:
```suggestion
image: nf-core/gitpod

vscode:
  extensions:
    - codezombiech.gitignore
    # - cssho.vscode-svgviewer
    - davidanson.vscode-markdownlint
    - eamodio.gitlens 
    - EditorConfig.EditorConfig
    - Gruntfuggly.todo-tree
    - mechatroner.rainbow-csv
    # - nextflow.nextflow
    - oderwat.indent-rainbow
    - redhat.vscode-yaml
    - streetsidesoftware.code-spell-checker
```

Reply:
The ones commented out are not yet available on openVS. svgviewer might not be so important, nextflow on the other hand would be 🍬

Reply:
Can we add some comments after each extension to say what they're adding and why? I see lots of extensions here and I'm wondering why it's needed.

Reply:
I added a comment to explain that the are based on https://marketplace.visualstudio.com/items?itemName=nf-core.nf-core-extensionpack